ppr-ui/src/components/unitNotes/UnitNotePanel.vue

```diff
@@ -114,41 +106,46 @@ export default defineComponent({
     UnitNoteHeaderInfo,
     UnitNoteContentInfo
   },
-  setup () {
+  setup (props) {
     const router = useRouter()
 
     const {
       setMhrUnitNoteType
     } = useStore()
 
     const {
-      isNoticeOfCautionOrRelatedDocType
+      getNoteOptions
     } = useMhrUnitNote()
 
+    const localState = reactive({
+      noteOptions: getNoteOptions(props.note)
```

Comment:
If you are not mutating locally, i'm not sure if this is required, or was there type errors on trying to access`.length` on the function in the template?

Could probably get away with using `getNoteOptions(props.note)` directly in the script or `getNoteOptions(note)` in the template. 

That being said, i'm impartial, if this is what you prefer!

Reply:
Yeah I can put it directly in the script, its not mutated. I rather call it in the script rather than template, so I won't need to call it multiple times.